src/main/index.js

```diff
@@ -68,6 +68,7 @@ app.on('ready', async () => {
         dialog.showErrorBox(
           'Daemon has Exited',
           'The daemon may have encountered an unexpected error, or another daemon instance is already running.'
+          <link rel="Read our FAQ" href="https://lbry.io/faq/startup-troubleshooting " />
```

Comment:
You should replace this with the actual url:
 `'https://lbry.io/faq/startup-troubleshooting'`